test/adapter/json/key_case_test.rb

```diff
@@ -21,21 +21,19 @@ class PostSerializer < ActiveModel::Serializer
         def setup
           ActionController::Base.cache_store.clear
           @blog = Blog.new(id: 1, name: 'My Blog!!', special_attribute: 'neat')
-          serializer = CustomBlogSerializer.new(@blog)
-          @adapter = ActiveModelSerializers::Adapter::Json.new(serializer)
         end
 
         def test_key_transform_default
           mock_request
           assert_equal({
             blog: { id: 1, special_attribute: 'neat', articles: nil }
-          }, @adapter.serializable_hash(@options))
+          }, adapter.serializable_hash)
```

Comment:
The `serialization_context` is no more passed via `as_json` options but via the adapter options.